pkg/sidecar/k8s_network.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+const (
+	defaultDataNetwork = "default"
```

Comment:
I think we can afford to make this public and add godoc.

Reply:
No need for this to be public - we need it public on the SDK side only, no package is currently using this particular one.